lib/pages/chat_list/chat_list.dart

```diff
@@ -826,6 +829,12 @@ class ChatListController extends State<ChatList>
               // label: space
               //     .getLocalizedDisplayname(MatrixLocals(L10n.of(context)!)),
               label: space.nameIncludingParents(context),
+              textStyle: TextStyle(
+                color: (firstSelectedRoom == null ||
```

Comment:
What are the circumstances when these conditions would happen? I don't think the add to space option is shown to non-admins

Reply:
You can't open the Add to Space button from a space you aren't an admin of. However, you can see that space as an option in the Add to Space menu of another space.

Will said to not show recursion-causing spaces in the Add to Space menu, but to show don't-have-permission spaces as grayed out + showing error if the user clicks them.

Reply:
Ah, that makes sense. Can you make a note of that in the comments?, then I'll merge this PR